### ISSUE TITLE
cudaCheckError() failed : invalid device function

### ISSUE THREAD

**Issue by hunter1987912** [jwyang/faster-rcnn.pytorch#224]
hi @jwyang 
I have a error, cudaCheckError() failed : invalid device function 
 can you help me? thanks~
---------------------------------------------------------------------------------------------------
cuda 8.0 , GTX 1080, 
CUDA_ARCH=-gencode arch=compute_61,code=sm_61
nvcc fatal   : Unsupported gpu architecture 'compute_61'
--------------------------------------------------------------------------------------------------
Loaded dataset `voc_2007_trainval` for training
Set proposal method: gt
Appending horizontally-flipped training examples...
voc_2007_trainval gt roidb loaded from /home/mrt600/Iven/liz/pytorchFasterRCNN/data/cache/voc_2007_trainval_gt_roidb.pkl
done
Preparing training data...
done
before filtering, there are 10022 images...
after filtering, there are 10022 images...
10022 roidb entries
Loading pretrained weights from data/pretrained_model/vgg16_caffe.pth
/home/mrt600/Iven/liz/pytorchFasterRCNN/lib/model/rpn/rpn.py:68: UserWarning: Implicit dimension choice for softmax has been deprecated. Change the call to include dim=X as an argument.
  rpn_cls_prob_reshape = F.softmax(rpn_cls_score_reshape)
cudaCheckError() failed : invalid device function

**Comment by jwyang** [comment]
@hunter1987912 you need to make sure your are using the right arch to compile your cuda code. Please read carefully the readme.

**Comment by rui1996** [comment]
I have similar problem. Mine is cuda 9.1 and GTX 1080, but when using "CUDA_ARCH=-gencode arch=compute_61,code=sm_61" as suggested in the Readme, it says unsupported. Why?
